src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -94,6 +100,13 @@ class ClusterResourceScheduler {
                                             bool requires_object_store_memory,
                                             bool *is_infeasible);
 
+  scheduling::NodeID GetBestSchedulableNode(const TaskSpecification &task_spec,
```

Comment:
There are already many a methods in this class. Can you directly append a default parameter (scheduling::NodeID forward_to = scheduling::NodeID::Nil()) to the existing method instead of adding a new method?